acl_loader/main.py

```diff
@@ -409,10 +417,17 @@ def convert_l2(self, table_name, rule_idx, rule):
             else:
                 try:
                     rule_props["ETHER_TYPE"] = int(rule.l2.config.ethertype)
-                except:
-                    raise AclLoaderException("Failed to convert ethertype %s table %s rule %s" % (
+                except Exception:
```

Comment:
>Exception [](start = 23, length = 9)

Not your fault. It ignores the original exception, we need at least log it or print to stderr.

Reply:
added!